### PR TITLE
Add @JorgeCastilloPrz Functional Android video

### PR DESCRIPTION
**Pull request by ersin-ertan**

### CODE REVIEW

File: kategory-docs/docs/docs/quickstart/blogs/README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ Introductory talk to Functional architectures to be built on top of Kategory.
 
 KotlinConf, San Francisco, Nov '17
 
+### Functional approach to Android architecture using Kotlin
+
+Move side effects to the edges of the system, implement a functional oriented architecture for Android apps based on purity.
+
+<iframe width="560" height="315" src="https://https://www.youtube.com/watch?v=qGef3sFAIxU?rel=0" frameborder="0" allowfullscreen></iframe>
```
Comment (tonilopezmr) [inline kategory-docs/docs/docs/quickstart/blogs/README.md]: fix the double `https://` in the URL please

**Comment by pakoito** [comment]
Please do not merge until the refactor for Arrow is complete.

**Comment by tonilopezmr** [comment]
Yes I know @pakoito. I only approve this PR, thanks for the reminder!

**Comment by raulraja** [comment]
@ersin-ertan can you update your PR with latest from master so we can merge it? thanks!

**Pull request closed without merging**